### PR TITLE
Add gh-pages

### PR DESCRIPTION
**Pull request by AndyMJones**

### CODE REVIEW

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,201 @@
+GEM
```
Comment (BenLambertNcl) [inline docs/Gemfile.lock]: Don't think we should need to track this file?

**Review by BenLambertNcl — APPROVED** [review]
LGTM :+1:, Just one file that doesn't need tracking I think

**Comment by ColinEberhardt** [comment]
LGTM - although does this content now duplicate some of the documentation in the `api-specification` package?

**Comment by BenLambertNcl** [comment]
I have a PR ready to go in once this is merged to copy the docs folder into each of the API demos, and remove the docs from `api-specification`.

**Pull request merged**